libs/platform/src/lib/components/dynamic-page/dynamic-page-content/dynamic-page-tabbed-content.component.ts

```diff
@@ -0,0 +1,150 @@
+    }
+
+    /**
+     * @hidden
+     * used internally to set margin top correctly for the content
+     */
+    @Input()
+    set contentTop(height: string) {
+        if (height) {
+            this._height = height;
+        }
+    }
+    get contentTop(): string {
+        return this._height;
+    }
+
+    /**
+     * @hidden
+     * tracking the background value
+     */
+    private _background: DynamicPageBackgroundType;
+
+    /**
+     * @hidden
+     * tracks the size for responsive padding
+     */
+    private _size: DynamicPageResponsiveSize;
+
+    /**
+     * @hidden
+     * tracks the height attribute
+     */
+    private _height: string;
+
+    constructor(
+        public _elementRef: ElementRef<HTMLElement>,
+        public _renderer: Renderer2,
+        public scrollDispatcher: ScrollDispatcher,
+        public zone: NgZone,
+        public _dynamicPageService: DynamicPageService
+    ) {}
+
+    /**
+     * @hidden
+     * sets the style classes for background property
+     * @param background
+     */
+    private _setBackgroundStyles(background: DynamicPageBackgroundType): void {
+        const hostElement = this._elementRef.nativeElement.querySelector('.fd-dynamic-page__content');
+        switch (background) {
+            case 'transparent':
+                this._addClassNameToCustomElement(hostElement, CLASS_NAME.dynamicPageContentTransparentBg);
+                break;
+            case 'list':
+                this._addClassNameToCustomElement(hostElement, CLASS_NAME.dynamicPageContentListBg);
+                break;
+            case 'solid':
+            default:
+                this._removeClassNameToHostElement(CLASS_NAME.dynamicPageContentTransparentBg);
+                this._removeClassNameToHostElement(CLASS_NAME.dynamicPageContentListBg);
+                break;
+        }
+    }
+
+    /**
+     * @hidden
+     * sets the padding classes
+     * @param sizeType
+     */
+    private _setSize(sizeType: DynamicPageResponsiveSize): void {
+        const hostElement = this._elementRef.nativeElement.querySelector('.fd-dynamic-page__content');
+
+        switch (sizeType) {
+            case 'small':
+                this._addClassNameToCustomElement(hostElement, CLASS_NAME.dynamicPageContentAreaSmall);
+                break;
+            case 'medium':
+                this._addClassNameToCustomElement(hostElement, CLASS_NAME.dynamicPageContentAreaMedium);
+                break;
+            case 'large':
+                this._addClassNameToCustomElement(hostElement, CLASS_NAME.dynamicPageContentAreaLarge);
+                break;
+            case 'extra-large':
+            default:
+                this._addClassNameToCustomElement(hostElement, CLASS_NAME.dynamicPageContentAreaExtraLarge);
+                break;
+        }
+    }
+    /**
+     * get reference to this element
+     */
+    getElementRef(): ElementRef<HTMLElement> {
+        return this._elementRef;
+    }
```

Comment:
Can you correct order of methods in this file?

Reply:
corrected